### PR TITLE
bitos project init

### PR DESCRIPTION
**Pull request by guoshijiang**
bitos project init

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+FROM golang:stretch
+
+RUN apt-get update -y
+RUN apt-get install ca-certificates jq -y
+
+WORKDIR /root
+
+COPY --from=build-env /go/src/github.com/bitdao-io/bitos/build/bitosd /usr/bin/bitosd
+
+EXPOSE 26656 26657 1317 9090
```
Comment (chengwenxi) [inline Dockerfile:21]: Expose 8545 for ETH json-rpc server?

**Comment by bradyjoestar** [comment]
It seems that `make protoc-gen` failed? The error log:
```
fatal: 没有发现名称，无法描述任何东西。
Generating Protobuf files
/usr/bin/docker run --rm -v /home/wenbin/go/src/github.com/bitdao-io/bitos:/workspace --workdir /workspace tendermintdev/sdk-proto-gen sh ./scripts/protocgen.sh
Unable to find image 'tendermintdev/sdk-proto-gen:latest' locally
latest: Pulling from tendermintdev/sdk-proto-gen
3aa4d0bbde19: Pull complete 
48ae170c2a8c: Pull complete 
cb35b180f419: Pull complete 
367011ddbc17: Pull complete 
318840287a89: Pull complete 
d1faf07dae44: Pull complete 
235be7043e23: Pull complete 
a3413cbf3dc8: Pull complete 
59bd1e37d8aa: Pull complete 
de338dd23037: Pull complete 
c1af6e2e67fb: Pull complete 
370c383c6f51: Pull complete 
ae1db572790c: Pull complete 
Digest: sha256:339931ac87ad7534f567a83ef140ec7815de05172d4aef927d6c9f391b6c49ad
Status: Downloaded newer image for tendermintdev/sdk-proto-gen:latest
2022/07/11 09:33:06 open ./docs/protodoc-markdown.tmpl: no such file or directory
--doc_out: exit status 1
make: *** [Makefile:458：proto-gen] 错误 1
```

**Comment by guoshijiang** [comment]
> protoc-gen

ok, I will fix it

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+## Reference
+
+- **[Users](./users/)**: Documentation on relevant technical concepts for users like `accounts`, `gas`, `tokens` or `transactions` on bitos as well as more advanced topics.
+- **[Developers](./developers/)**: Documentation on `block explorers`, `clients`, the `JSON-RPC` implementation or the testnet `faucet` can be found here.
+- **[Validators](./validators/)**: Documentation on how to become an bitos validator, hardware requirements, configuring a `node` and connecting to the `testnet` and `mainnet`,
+
+## Other Resources
+
+- **[Module Directory](../x/)**: Module implementations and their respective documentation.
+- **[Ethermint API Reference](https://pkg.go.dev/github.com/bitos/ethermint)**: Godocs of Ethermint.
```
Comment (bradyjoestar) [inline docs/README.md:50]: shoule we change it to `evmos/ethermint` or `bitdao-io/ethermint`?

Comment (guoshijiang) [reply docs/README.md]: maybe not need

Comment (guoshijiang) [reply docs/README.md]: We haven't fork ethermint yet

**Review by chengwenxi — APPROVED** [review]

**Review by FletcherMan — APPROVED** [review]

**Pull request merged**